### ISSUE TITLE
Suggestion: Add host to useFunctionsEmulator

### ISSUE THREAD

**Issue by shiftgeist** [nuxt-community/firebase-module#272]
### Is your feature request related to a problem? Please describe.
I'm working with the emulator and have not found a way to not use localhost as the host for functions. I am working on seperate machines and can't connect to the emulator of the other machine (in the same network).

### Describe the solution you'd like
I suggest to add a host option like `emulatorPort`, it could be in the same settings object: `firebase.services.functions.emulatorHost` (eg `10.10.10.3`).

### Describe alternatives you've considered
Only other way I found is not to use the firebase-module and manually setup firebase in nuxt, which can be quite frustrating and is far more difficult than using this module.

### Additional context
Example config:
```
firebase: {
    config: {
        [...]
    },
    services: {
      functions: {
        location: 'europe-west3',
        emulatorHost: '10.10.10.3',
        emulatorPort: 5001,
      },
    },
  },
```

Related issue (`emulatorPort`): https://github.com/nuxt-community/firebase-module/issues/64

Most likely related code: [lib/plugins/main.js:242](https://github.com/nuxt-community/firebase-module/blob/dev/lib/plugins/main.js#L242)

**Comment by lupas** [comment]
Thanks for the suggestion!
Will have a look at it, hopefully I'll find some time in September, if it's urgent feel free to do a PR.

**Comment by lupas** [comment]
Hey @shiftgeist 

This has been implemented with https://github.com/nuxt-community/firebase-module/commit/08db302013fdac1213e783f6cd860ca62cc15cbe and will be shipped with v7.0.0 in the coming days.

Thanks for the issue!
